Cost Analysis Documentation (#13739)
open-metadata/OpenMetadata: openmetadata-docs/content/v1.2.x/features/data-insight/cost-analaysis.md

---
title: Cost Analysis
slug: /features/data-insight/cost-analysis
---

# Cost Analysis (Collate Only)

> Gain valuable insights into your resource allocation and utilization with `Cost Analysis`. With a suite of interactive graphs, it empowers you to make data-driven decisions to optimize your costs and improve your resource management. You'll be equipped to make informed decisions, reduce wastage, and optimize your resources, ultimately enhancing cost-efficiency and performance.

Reviewer: ```suggestion
Gain valuable insights into your resource allocation and utilization with `Cost Analysis`. With a suite of interactive graphs, it empowers you to make data-driven decisions to optimize your data platform costs and improve your resource management. You'll be equipped to make informed decisions, and optimize your resources.
```

## Supported Connectors
{% multiTablesWrapper %}

| Connector          | Status                       |
| :----------------- | :--------------------------- |
| Snowflake          | {% icon iconName="check" /%} |

{% /multiTablesWrapper %}

## Requirements

> For the supported connectors, below workflows are required to be executed after the metadata ingestion is completed to gather the necessary data for the Cost Analysis Charts.

Reviewer: ```suggestion
To have cost analysis data available you will need to execute the below workflows are after the metadata ingestion has run.
```

1. **Usage Workflow**:
- Purpose: Gather lifecycle information for data assets.

> - Description: The Usage Workflow collects essential information regarding data asset Life Cycle. This information includes details such as `accessed_at` timestamps and `accessed_by` user information for each data asset. This data is crucial for determining whether a data asset is frequently used or unused within a specified time period.

Reviewer: ```suggestion
- Description: The Usage Workflow collects essential information regarding the data asset Life Cycle. This information includes details such as `accessed_at` timestamps and `accessed_by` user information for each data asset. This data is crucial to determine whether a data asset is frequently used or unused within a specified time period.
```
